integration/reuse_layer_rebuild_test.go

```diff
@@ -164,142 +164,147 @@ func testReusingLayerRebuild(t *testing.T, context spec.G, it spec.S) {
 			containerIDs[secondContainer.ID] = struct{}{}
 
 			Eventually(secondContainer).Should(BeAvailable())
-			Eventually(secondContainer).Should(Serve(MatchRegexp(`Hello from Ruby 2\.7\.\d+`)).OnPort(8080))
+			Eventually(secondContainer).Should(Serve(MatchRegexp(`Hello from Ruby 3\.1\.\d+`)).OnPort(8080))
 
 			Expect(secondImage.Buildpacks[0].Layers["mri"].SHA).To(Equal(firstImage.Buildpacks[0].Layers["mri"].SHA))
 		})
 	})
 
-	context("when an app is rebuilt and there is a change", func() {
-		it("rebuilds the layer", func() {
-			var (
-				err         error
-				logs        fmt.Stringer
-				firstImage  occam.Image
-				secondImage occam.Image
-
-				firstContainer  occam.Container
-				secondContainer occam.Container
-			)
-
-			source, err = occam.Source(filepath.Join("testdata", "simple_app"))
-			Expect(err).NotTo(HaveOccurred())
-
-			build := pack.Build.
-				WithPullPolicy("never").
-				WithBuildpacks(
-					settings.Buildpacks.MRI.Online,
-					settings.Buildpacks.BuildPlan.Online,
+	// This test is not currently applicable on jammy because currently Jammy support
+	// only applies to one version of Ruby (version 3.1 and above)
+	// This condition can be removed when multiple versions of Ruby are supported on Jammy (e.g. 3.0 or 3.2)
+	if builder.BuilderName != "paketobuildpacks/builder-jammy-buildpackless-base" {
```

Comment:
I think this guard should be more inclusive of any builder that isn't based on a Jammy stack.

```suggestion
	if builder.LocalInfo.Stack != "io.buildpacks.stacks.jammy" {
```